app/components/support_interface/provider_user_summary_component.rb

```diff
@@ -0,0 +1,47 @@
+module SupportInterface
+  class ProviderUserSummaryComponent < ViewComponent::Base
+    include ViewHelper
+
+    attr_reader :provider_user
+
+    def initialize(provider_user)
+      @provider_user = provider_user
+    end
```

Comment:
Can we change this to inherit from the SummaryListComponent and remove the template?

I think similar to this...
```suggestion
  class ProviderUserSummaryComponent < SummaryListComponent
    include ViewHelper

    attr_reader :provider_user
    
    def initialize(provider_user)
      @provider_user = provider_user
      super(table_rows)
    end
 
```